docs/source/certs_management.md

```diff
@@ -0,0 +1,382 @@
+
+**Enrollment** – A public/private key pair and an X.509 certificate issued by the organization's Certificate Authority (CA). The certificate encodes roles, attributes, and metadata, which represent an identity in a Fabric network. An enrollment is associated with a CA registration by username and password.
+
+**Identity** - A public certificate and its private key used for encryption. The public certificate is the X.509 certificate issued by the CA, while the private key is stored out of band, on a secure storage.
+
+**TLS** - A public Transport Layer Security (TLS) Certificate that authorizes client and node communications. On Fabric, registration and enrollment are the same for X.509 Certificates and TLS Certificates.
+
+
+## Certificate Types
+
+Hyperledger Fabric implements two types of certificates: 1) **Enrollment** Certificates for identities and 2) **TLS** Certificates for node and client communications.
+
+### Enrollment Certificates
+
+Enrollment Certificates are classed into four types:
+
+* **Admin**
+* **Peer**
+* **Orderer**
+* **Client**
+
+Each Enrollment Certificate type has a specific role:
+
+**Admin:** X.509 Certificates used to authenticate admin identities, which are required to make changes to  Fabric configurations.
+
+**Peer:** X.509 Certificates used to enroll peer nodes, located physically on the node or mapped to the node. For a Fabric peer node to start, it must have a valid Enrollment Certificate with the required attributes.
+
+**Orderer:** X.509 Certificates used to enroll orderer nodes, located physically on the node or mapped to the node. For a Fabric orderer node to start, it must have a valid Enrollment Certificate with the required attributes.
+
+**Client:** X.509 Certificates that allow signed requests to be passed from clients to Fabric nodes. Client certs define the identities of client applications submitting transactions to a Fabric network.
+
+
+### TLS Certificates
+
+TLS Certificates allow Fabric nodes and clients to sign and encrypt communications. A valid TLS Certificate is required for any channel communication.
+
+### Certificate Expiration
+
+Enrollment and TLS Certificates are assigned an expiration date by the issuing Certificate Authority (CA).  Expiration dates must be monitored, and certificates must be re-enrolled before expiration. The most important certificate parameter is the **Not After** element, which indicates its expiration date.
+
+
+## Certificates and Locations
+
+Organization CAs supply X.509 Enrollment Certificates for identities and the TLS CAs supply TLS Certificates for securing node and client communications.
+
+
+### Organization CA Certificates
+
+Organization CA Root Certificates and Organization CA Admin Certificates are self-signed certificates if creating a new Certificate Authority (CA), or provided by an external CA.
```

Comment:
This sentence is only true for Organization CA Root Certificates, not Organization CA Admin Certificates. Therefore I would move this sentence to the Description of the next section.

Reply:
ok. added some content to replace it to avoid an empty level 3 heading.